cluster/gke.go

```diff
@@ -321,6 +322,10 @@ func (c *GKECluster) CreateCluster() error {
 		// TODO status code !?
 		return errors.New(be.Message)
 	}
+	if gkeCluster.Status == statusError {
```

Comment:
in which cases will `err == nil` be true (i.e. the function not returning in line 323), but the error extraction below work?

Reply:
If getClusterGoogle func return without any error and the gkeCluster.Status is ERROR.
`gkeCluster, err := getClusterGoogle(svc, cc)`